### DISCUSSION TITLE
Donation for contributors

### DISCUSSION THREAD

**Discussion by Matthias1111** [torbennehmer/hacs-e3dc#187] [Ideas]
Hi guys, 

I really like this integration and use is it since about one year. Is there any option to donate "a coffee" or something like that for you great work. I just want to say thank you. :-)

Best Regards
Matthias

**Comment by torbennehmer** [comment]
@Matthias1111 Sorry for the late answer. As you can see, my real life has the better of me these days.
It's a pleasure and great to hear that you're using the integration.
I don't have a donation button, probably won't introduce one too, if only for tax reasons here in Germany ;-). 
It'd be a much greater help if you'd contribute, I take everything, not only code contribution but also support, documentation, etc. 
It's a joint endavour for me, not something, I'd intent to make money with :)
Cheers, Torben
